src/openforms/js/components/admin/form_design/logic/actions/dmn/DMNActionConfig.stories.js

```diff
@@ -232,10 +233,16 @@ export const Empty = {
 
       const [formVarsDropdowns, dmnVarsDropdown] = dropdowns;
 
-      await userEvent.selectOptions(formVarsDropdowns, 'Name (name)');
       await userEvent.selectOptions(dmnVarsDropdown, 'camundaVar');
+      await selectEvent.select(formVarsDropdowns, (content, element) => {
+        return (
+          content === '(name)' && element.classList.contains('form-variable-dropdown__option__key')
+        );
+      });
```

Comment:
DOM queries go against the spirit of testing-library - if there's no other way, I would at least expect this workaround to be abstract in a helper like `selectFormVariable(...)`

Reply:
I've replaced this function with a simpeler 'matcher'.

But i've also added `storybookTestHelpers.js` for some helper functions 👍